### PR TITLE
add: links to africa cdc pgi builds

### PR DESCRIPTION
**Pull request by j23414**
### Description of proposed changes    

Update the https://nextstrain.org/sars-cov-2/ map to link out to the country-level analyses at https://nextstrain.org/groups/africa-cdc. 

Mostly following steps from: https://docs.nextstrain.org/en/latest/guides/share/sars-cov-2.html

### Testing

Check links. Let me know if we're allowed to have duplicate dataset entries for the same location or if I need unique geo IDs.

Example:

```
  - url: https://nextstrain.org/groups/SC2ZamPub/ncov/africa/zambia
    name: Zambia
    geo: zambia
    level: country
    coords:
      - 30
      - -15
    org:
      name: Zambia Genomic Sequencing Consortium
      url: https://nextstrain.org/groups/SC2ZamPub
```

and 

```
  - url: https://nextstrain.org/groups/africa-cdc/ncov/zambia?c=clade_membership&f_country=Zambia&p=grid&tl=country
    name: Zambia
    geo: zambia
    level: country
    coords:
      - 27.5599164      #<= pulled from lat_longs.tsv, not sure if these need to exactly match the earlier entry
      - -14.5186239
    org:
      name: Africa CDC PGI
      url: https://africacdc.org/institutes/ipg/
```

I'm open to suggestions or edits.

### CODE REVIEW

**Comment by joverlee521** [comment]
> Let me know if we're allowed to have duplicate dataset entries for the same location or if I need unique geo IDs.

Totally fine to have multiple datasets for the same location. In the past, the [coordinates have been tweaked to prevent datasets from overlapping on the map](https://github.com/nextstrain/nextstrain.org/commit/dbb5708153e97f767da77cfcb3b013a6656730f6).

**Comment by j23414** [comment]
For checking links:

```
grep "url" static-site/content/SARS-CoV-2-Datasets.yaml  |\
  grep "africa-cdc" |\
  sed 's/- url: //g' |\
  awk -F'https://nextstrain.org/groups/africa-cdc/ncov/' 'OFS="|" {print $2,$0}' |\
  sed 's/?c=clade_membership.*tl=country|//g' |\
  awk '{print "* [ ] ["$1"]("$2")"}' > links.md
```

which will create a list of links:

* [ ] [central-africa](https://nextstrain.org/groups/africa-cdc/ncov/central-africa?c=clade_membership&f_country=Burundi,Cameroon,Central%20African%20Republic,Democratic%20Republic%20of%20the%20Congo,Equatorial%20Guinea,Gabon,Republic%20of%20the%20Congo,Chad&p=grid&tl=country)
* [ ] [eastern-africa](https://nextstrain.org/groups/africa-cdc/ncov/eastern-africa?c=clade_membership&f_country=Comoros,Djibouti,Ethiopia,Kenya,Madagascar,Mauritius,Rwanda,Seychelles,Somalia,South%20Sudan,Sudan,Uganda&p=grid&tl=country)
* [ ] [northern-africa](https://nextstrain.org/groups/africa-cdc/ncov/northern-africa?c=clade_membership&f_country=Algeria,Egypt,Libya,Morocco,Tunisia&p=grid&tl=country)
* [ ] [southern-africa](https://nextstrain.org/groups/africa-cdc/ncov/southern-africa?c=clade_membership&f_country=Angola,Botswana,Eswatini,Lesotho,Malawi,Mozambique,Namibia,South%20Africa,Zambia,Zimbabwe&p=grid&tl=country)
* [ ] [western-africa](https://nextstrain.org/groups/africa-cdc/ncov/western-africa?c=clade_membership&f_country=Benin,Burkina%20Faso,Cabo%20Verde,Cote%20d%27Ivoire,Gambia,Ghana,Guinea,Guinea-Bissau,Liberia,Mali,Niger,Nigeria,Senegal,Sierra%20Leone,Togo&p=grid&tl=country)
* [ ] [burundi](https://nextstrain.org/groups/africa-cdc/ncov/burundi?c=clade_membership&f_country=Burundi&p=grid&tl=country)
* [ ] [cameroon](https://nextstrain.org/groups/africa-cdc/ncov/cameroon?c=clade_membership&f_country=Cameroon&p=grid&tl=country)
* [ ] [central-african-republic](https://nextstrain.org/groups/africa-cdc/ncov/central-african-republic?c=clade_membership&f_country=Central%20African%20Republic&p=grid&tl=country)
* [ ] [chad](https://nextstrain.org/groups/africa-cdc/ncov/chad?c=clade_membership&f_country=Chad&p=grid&tl=country)
* [ ] [republic-of-the-congo](https://nextstrain.org/groups/africa-cdc/ncov/republic-of-the-congo?c=clade_membership&f_country=Republic%20of%20the%20Congo&p=grid&tl=country)
* [ ] [democratic-republic-of-the-congo](https://nextstrain.org/groups/africa-cdc/ncov/democratic-republic-of-the-congo?c=clade_membership&f_country=Democratic%20Republic%20of%20the%20Congo&p=grid&tl=country)
* [ ] [equatorial-guinea](https://nextstrain.org/groups/africa-cdc/ncov/equatorial-guinea?c=clade_membership&f_country=Equatorial%20Guinea&p=grid&tl=country)
* [ ] [gabon](https://nextstrain.org/groups/africa-cdc/ncov/gabon?c=clade_membership&f_country=Gabon&p=grid&tl=country)
* [ ] [comoros](https://nextstrain.org/groups/africa-cdc/ncov/comoros?c=clade_membership&f_country=Comoros&p=grid&tl=country)
* [ ] [djibouti](https://nextstrain.org/groups/africa-cdc/ncov/djibouti?c=clade_membership&f_country=Djibouti&p=grid&tl=country)
* [ ] [ethiopia](https://nextstrain.org/groups/africa-cdc/ncov/ethiopia?c=clade_membership&f_country=Ethiopia&p=grid&tl=country)
* [ ] [kenya](https://nextstrain.org/groups/africa-cdc/ncov/kenya?c=clade_membership&f_country=Kenya&p=grid&tl=country)
* [ ] [madagascar](https://nextstrain.org/groups/africa-cdc/ncov/madagascar?c=clade_membership&f_country=Madagascar&p=grid&tl=country)
* [ ] [mauritius](https://nextstrain.org/groups/africa-cdc/ncov/mauritius?c=clade_membership&f_country=Mauritius&p=grid&tl=country)
* [ ] [rwanda](https://nextstrain.org/groups/africa-cdc/ncov/rwanda?c=clade_membership&f_country=Rwanda&p=grid&tl=country)
* [ ] [seychelles](https://nextstrain.org/groups/africa-cdc/ncov/seychelles?c=clade_membership&f_country=Seychelles&p=grid&tl=country)
* [ ] [somalia](https://nextstrain.org/groups/africa-cdc/ncov/somalia?c=clade_membership&f_country=Somalia&p=grid&tl=country)
* [ ] [south-sudan](https://nextstrain.org/groups/africa-cdc/ncov/south-sudan?c=clade_membership&f_country=South%20Sudan&p=grid&tl=country)
* [ ] [sudan](https://nextstrain.org/groups/africa-cdc/ncov/sudan?c=clade_membership&f_country=Sudan&p=grid&tl=country)
* [ ] [uganda](https://nextstrain.org/groups/africa-cdc/ncov/uganda?c=clade_membership&f_country=Uganda&p=grid&tl=country)
* [ ] [algeria](https://nextstrain.org/groups/africa-cdc/ncov/algeria?c=clade_membership&f_country=Algeria&p=grid&tl=country)
* [ ] [egypt](https://nextstrain.org/groups/africa-cdc/ncov/egypt?c=clade_membership&f_country=Egypt&p=grid&tl=country)
* [ ] [libya](https://nextstrain.org/groups/africa-cdc/ncov/libya?c=clade_membership&f_country=Libya&p=grid&tl=country)
* [ ] [morocco](https://nextstrain.org/groups/africa-cdc/ncov/morocco?c=clade_membership&f_country=Morocco&p=grid&tl=country)
* [ ] [tunisia](https://nextstrain.org/groups/africa-cdc/ncov/tunisia?c=clade_membership&f_country=Tunisia&p=grid&tl=country)
* [ ] [angola](https://nextstrain.org/groups/africa-cdc/ncov/angola?c=clade_membership&f_country=Angola&p=grid&tl=country)
* [ ] [botswana](https://nextstrain.org/groups/africa-cdc/ncov/botswana?c=clade_membership&f_country=Botswana&p=grid&tl=country)
* [ ] [eswatini](https://nextstrain.org/groups/africa-cdc/ncov/eswatini?c=clade_membership&f_country=Eswatini&p=grid&tl=country)
* [ ] [lesotho](https://nextstrain.org/groups/africa-cdc/ncov/lesotho?c=clade_membership&f_country=Lesotho&p=grid&tl=country)
* [ ] [malawi](https://nextstrain.org/groups/africa-cdc/ncov/malawi?c=clade_membership&f_country=Malawi&p=grid&tl=country)
* [ ] [mozambique](https://nextstrain.org/groups/africa-cdc/ncov/mozambique?c=clade_membership&f_country=Mozambique&p=grid&tl=country)
* [ ] [namibia](https://nextstrain.org/groups/africa-cdc/ncov/namibia?c=clade_membership&f_country=Namibia&p=grid&tl=country)
* [ ] [south-africa](https://nextstrain.org/groups/africa-cdc/ncov/south-africa?c=clade_membership&f_country=South%20Africa&p=grid&tl=country)
* [ ] [zambia](https://nextstrain.org/groups/africa-cdc/ncov/zambia?c=clade_membership&f_country=Zambia&p=grid&tl=country)
* [ ] [zimbabwe](https://nextstrain.org/groups/africa-cdc/ncov/zimbabwe?c=clade_membership&f_country=Zimbabwe&p=grid&tl=country)
* [ ] [benin](https://nextstrain.org/groups/africa-cdc/ncov/benin?c=clade_membership&f_country=Benin&p=grid&tl=country)
* [ ] [burkina-faso](https://nextstrain.org/groups/africa-cdc/ncov/burkina-faso?c=clade_membership&f_country=Burkina%20Faso&p=grid&tl=country)
* [ ] [cabo-verde](https://nextstrain.org/groups/africa-cdc/ncov/cabo-verde?c=clade_membership&f_country=Cabo%20Verde&p=grid&tl=country)
* [ ] [cote-divoire](https://nextstrain.org/groups/africa-cdc/ncov/cote-divoire?c=clade_membership&f_country=Cote%20d%27Ivoire&p=grid&tl=country)
* [ ] [gambia](https://nextstrain.org/groups/africa-cdc/ncov/gambia?c=clade_membership&f_country=Gambia&p=grid&tl=country)
* [ ] [ghana](https://nextstrain.org/groups/africa-cdc/ncov/ghana?c=clade_membership&f_country=Ghana&p=grid&tl=country)
* [ ] [guinea](https://nextstrain.org/groups/africa-cdc/ncov/guinea?c=clade_membership&f_country=Guinea&p=grid&tl=country)
* [ ] [guinea-bissau](https://nextstrain.org/groups/africa-cdc/ncov/guinea-bissau?c=clade_membership&f_country=Guinea-Bissau&p=grid&tl=country)
* [ ] [liberia](https://nextstrain.org/groups/africa-cdc/ncov/liberia?c=clade_membership&f_country=Liberia&p=grid&tl=country)
* [ ] [mali](https://nextstrain.org/groups/africa-cdc/ncov/mali?c=clade_membership&f_country=Mali&p=grid&tl=country)
* [ ] [niger](https://nextstrain.org/groups/africa-cdc/ncov/niger?c=clade_membership&f_country=Niger&p=grid&tl=country)
* [ ] [nigeria](https://nextstrain.org/groups/africa-cdc/ncov/nigeria?c=clade_membership&f_country=Nigeria&p=grid&tl=country)
* [ ] [senegal](https://nextstrain.org/groups/africa-cdc/ncov/senegal?c=clade_membership&f_country=Senegal&p=grid&tl=country)
* [ ] [sierra-leone](https://nextstrain.org/groups/africa-cdc/ncov/sierra-leone?c=clade_membership&f_country=Sierra%20Leone&p=grid&tl=country)
* [ ] [togo](https://nextstrain.org/groups/africa-cdc/ncov/togo?c=clade_membership&f_country=Togo&p=grid&tl=country)

File: static-site/content/SARS-CoV-2-Datasets.yaml [hunk 1]
```diff
@@ -1940,3 +2148,609 @@ datasets:
...
+      name: Africa CDC PGI
+      url: https://africacdc.org/institutes/ipg/
+
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/cameroon?c=clade_membership&f_country=Cameroon&p=grid&tl=country
+    name: Cameroon
+    geo: cameroon
+    level: country
+    coords:
+      - 13.1535811
+      - 4.6125522
```
Comment (huddlej) [inline static-site/content/SARS-CoV-2-Datasets.yaml:2225]: @j23414 Can you assign different lat/lng for this Cameroon build so it doesn't overlap with [Bryan Tegomoh's existing Cameroon build](https://github.com/nextstrain/nextstrain.org/blob/a78061708ffb23d15e596354835fe8229329e487/static-site/content/SARS-CoV-2-Datasets.yaml#L517-L526)?

Comment (j23414) [reply static-site/content/SARS-CoV-2-Datasets.yaml]: Fixed! Sorry I should have checked the [map](https://nextstrain-s-updatelink-p4ai8z.herokuapp.com/sars-cov-2/#datasets) for overlaps

**Comment by trvrb** [comment]
Looks generally great. One small thing. "Northern Africa" has the wrong lat/long location:

northern-africa

**Review by trvrb — APPROVED** [review]

**Pull request merged**